lib/monotonic_tick_count/version.rb

```diff
@@ -1,3 +1,3 @@
-module MonotonicTickCount
+class MonotonicTickCount
   VERSION = "0.1.0"
```

Comment:
Worth it to bump the version since it's changing so much?

Reply:
Actually this is brand new! I just showed the diff relative to the generated boilerplate.

Reply:
Ah, gotcha!